var/spack/repos/builtin/packages/mpark-variant/package.py

```diff
@@ -20,6 +20,7 @@ class MparkVariant(CMakePackage):
     # Ref.: https://github.com/mpark/variant/pull/73
     patch('nvcc.patch', when='@:1.4.0')
     patch('version.patch', when='@1.4.0')
+    patch('icpc.patch', when='@:1.4.0')
```

Comment:
Just so we can keep track of things:
```suggestion
    # Ref.: https://github.com/mpark/variant/issues/60
    patch('version.patch', when='@1.4.0')
    # Ref.: https://github.com/mpark/variant/pull/78
    patch('icpc.patch', when='@:1.4.0')
```